proto/rill/local/v1/api.proto

```diff
@@ -42,3 +51,56 @@ message GetVersionResponse {
   string current = 1;
   string latest = 2;
 }
+
+message DeployValidationRequest {}
+
+message DeployValidationResponse {
+  bool is_authenticated = 1; // if true below fields are relevant after login
+  bool is_github_connected = 2; // if true below fields are relevant after git install
+  string git_grant_access_url = 3; // redirect to this if is_github_connected or is_repo_access_granted is false
+  string git_user_name = 4;
```

Comment:
Is this the Git username or Github username? If the latter, use `github_username`

Reply:
github